use/web/admin-panel.md

```diff
@@ -69,6 +69,46 @@ Once the form and media are successfully uploaded, click on the add button to su
 
 ![media](../../../media/a11.png)
 
+#### Monitoring
+
+#### 1. Overview Section
+
+![media](../../../media/m1.png)
+
+#### Enter date in the date input which has an auto suggesting dropdown.
+
+![media](../../../media/m2.png)
+
+#### If data available
+
+![media](../../../media/m3.png)
+
+#### If no data available
+
+![media](../../../media/m4.png)
+
+#### 2. UCI-API
+
+#### If a date was already chosen in any of the services section or overview section, then bar chart(by default) would be shown based on the data available
+
+![media](../../../media/m5.png)
+
+#### If pie chart is chosen
+
+![media](../../../media/m6.png)
+
+#### If line chart is chosen
+
+![media](../../../media/m7.png)
```

Comment:
The screenshots below represent the different types of visualizations that are available:

- Bar Graph

   [IMAGE]

- Pie chart 
   
   [IMAGE]

- Line Graph

  [IMAGE]

Reply:
Please resolve the merge conflicts.